src/Shell/Task/SimpleMigrationTask.php

```diff
@@ -115,7 +115,9 @@ public function getOptionParser()
 
         $parser->description(
             'Bake migration class.'
-        )->addOption('plugin', [
+        )->addArgument('name', [
+            'help' => 'Name of the migration to bake. Can use Plugin.name to bake migration files into plugins.'
```

Comment:
You should also add the ``required`` option to the config array and set it to ``true``, since the command won't run if the ``name`` argument is not specified.